.gitignore

```diff
@@ -16,3 +16,6 @@ karma-xunit.xml
 checkstyle.xml
 .loopbackrc
 loopback-sdk-angular-*.tgz
+!intl/
+intl/*
+!intl/en/
```

Comment:
* Add an empty line at the end of file; 
* I think you don't need if you need `!intl/`, do you?